lucetc/src/options.rs

```diff
@@ -244,6 +247,12 @@ impl Options {
                      .takes_value(true)
                      .help("Path to the secret key to sign the object file. The file can be prefixed with \"raw:\" in order to store a raw, unencrypted secret key")
             )
+            .arg(
+                Arg::with_name("count_instructions")
+                    .long("--count_instructions")
```

Comment:
the arg should have a hyphen rather than an underscore; I dunno why our convention in this file has underscores in the name, but it's not as fun to type